Spring.FluentContext/Utils/ConfigHelper.cs

```diff
@@ -0,0 +1,24 @@
+using Spring.FluentContext.BuildingStages.Objects;
+using Spring.FluentContext.Definitions;
+using Spring.Validation;
+using Spring.Validation.Actions;
+
+namespace Spring.FluentContext.Utils
+{
+    public static class ConfigHelper
```

Comment:
Similarly, what is the purpose of this class?
I do not see any tests added nor class being used in the example project.

Could you please either delete it, or update tests, example and maybe leave some description so that I could update the wiki page?